features/admin-file.feature

```diff
@@ -0,0 +1,34 @@
+@admin-file @issue-457
+  Feature: An admin user can edit and delete attribute in admin file page
+    As an admin user
+    I can add/edit/save/delete attribute
```

Comment:
This feature is missing the last part:
`So that <something important to my job can happen>`

Maybe @only1chunts can provide his reason for wanting to be able to attach files with attributes?

Reply:
I guess you are looking for something like:
`So that I can associate various attributes to files`
Do that make sense in this context?
We add attributes like "License:GPLv3" or "PixelResolution:1024" etc the left part of the currie is from the Attributes table, the right part is the value.